README.md

```diff
@@ -20,12 +20,8 @@
 ### Prerequisites
 
 - git
-- Epic Games Launcher
-- Unreal Engine installed from the Epic Games Launcher; you will need every version that the plugin supports (currently 4.27, 5.0, 5.1, 5.2)
-- Visual Studio 2022 with C++ support
-- JetBrains Rider IDE (recommended for working with Unreal Engine C++ code)
-
-If not Rider, Visual Studio is fine and even better with the Resharper plugin.
+- Unreal Engine installed from the Epic Games Launcher or source code; Prior please check what Unreal Engine versions currently supported.
+- Microsoft Visual Studio. It's important that a compatible Visual Studio version should be installed. Please refer to Unreal Engine official documentations.
```

Comment:
Please refer to Unreal Engine's official documentation. [Can you link to the page about compatible Visual Studio versions?]